doc/source/reference/array_api.rst

```diff
@@ -5,14 +5,12 @@ Array API standard compatibility
 ********************************
 
 NumPy's main namespace as well as the `numpy.fft` and `numpy.linalg` namespaces
-are compatible [1]_ with the
-`2022.12 version <https://data-apis.org/array-api/2022.12/index.html>`__
+are compatible with the
+`2024.12 version <https://data-apis.org/array-api/2024.12/index.html>`__
 of the Python array API standard.
```

Comment:
Maybe `The NumPy 2.3.0 main ...` so that it is clear which NumPy version has compatibility.

Reply:
Done!